src/functions/index.ts

```diff
@@ -1,4 +1,9 @@
 export { default as createTimedCuration } from './create-timed-curation';
 export { default as listTimedCurations } from './list-timed-curations';
 export { default as updateTimedCuration } from './update-timed-curation';
-export { default as deleteTimedCuration } from './delete-timed-curation';
\ No newline at end of file
+export { default as deleteTimedCuration } from './delete-timed-curation';
+export { default as createScheduledCrawl } from './scheduled-crawls/create-scheduled-crawl';
+export { default as listScheduledCrawls } from './scheduled-crawls/list-scheduled-crawls';
+export { default as updateScheduledCrawl } from './scheduled-crawls/update-scheduled-crawl';
+export { default as deleteScheduledCrawl } from './scheduled-crawls/delete-scheduled-crawl';
+// TODO: Is this missing find-scheduled-crawl, find-timed-curation?
```

Comment:
Resolve this TODO (by adding missing functions)

Reply:
All missing functions added